Modules/Screenshot.cpp

```diff
@@ -0,0 +1,173 @@
+#include <X11/Xlib.h>
+#include <X11/Xutil.h>
+#include <png.h>
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <cstdlib>
+
+#include "Screenshot.h"
+#include "../include/json.hpp"
+#include "../Utils/CryptoUtils.h"
+
+
+// Function to convert image data to base64
+std::string Screenshot::base64_encode(unsigned char const* bytes_to_encode, unsigned int in_len) {
```

Comment:
change to the base64 encode function in the utils

Reply:
Done.